catalog-rest-service/src/main/java/org/openmetadata/catalog/CatalogApplicationDev.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one or more
+ *  contributor license agreements. See the NOTICE file distributed with
+ *  this work for additional information regarding copyright ownership.
+ *  The ASF licenses this file to You under the Apache License, Version 2.0
+ *  (the "License"); you may not use this file except in compliance with
+ *  the License. You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.openmetadata.catalog;
+
+import org.openmetadata.catalog.util.TablesInitializer;
+
+import java.util.List;
+
+/**
+ * Main catalog application
+ */
+public class CatalogApplicationDev {
+  public CatalogApplicationDev() {
+
+  }
+  public static void main(String[] args) throws Exception {
```

Comment:
This is assuming that devs always want to drop and create new tables.
They can also setup application config for TabelsInitializer. We can document this but don't want this to be the default way of setting dev setup. Lets remove this @amiorin

Reply:
Done